Twig/Extension/FormatterMediaExtension.php

```diff
@@ -76,14 +76,6 @@ public function getTwigExtension()
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function getName()
```

Comment:
So what would be a possible solution? Change `BaseProxyExtension` in SonataFormatterBundle not to implement `Twig_ExtensionInterface`?

[http://packanalyst.com/class?q=Sonata%5CFormatterBundle%5CExtension%5CBaseProxyExtension](http://packanalyst.com/class?q=Sonata%5CFormatterBundle%5CExtension%5CBaseProxyExtension)